docs/usage.md

```diff
@@ -98,6 +98,18 @@ $ cernopendata-client get-metadata --recid 329 --output-value authors.name --fil
 Rousseau, David
 ```
 
+Another example of using \--filter would be retrieving the container image
```

Comment:
Thanks, ported to Markdown that we switched to in the meantime.

Reply:
FWIW the CI tests are not passing due to a known issue that is being addressed in #147.